lnwire/channel_update.go

```diff
@@ -16,6 +16,14 @@ const (
 	// ChanUpdateRequiredMaxHtlc is a bit that indicates whether the
 	// required htlc_maximum_msat field is present in this ChannelUpdate.
 	ChanUpdateRequiredMaxHtlc ChanUpdateMsgFlags = 1 << iota
+
+	// ChanUpdateDontForward is a bit that indicates whether the
+	// ChanUpdate msg should be forwarded to other peers when
+	// received.
+	// NOTE: We currently do not check this bit when receiving a channel
+	// update from the peer to make sure we do not relay it because
+	// we already have other means (AuthProof) to do so.
```

Comment:
Does the respective AuthProof code need a comment update about this new `dont_forward` bit. Not sure if that makes sense, just asking.

Reply:
Yes had also the same thought, but honestly it would make the code look more confusing IMO this is the relevant code part:

https://github.com/lightningnetwork/lnd/blob/e549096b8837a95e0515c8c63fa01c634714d913/discovery/gossiper.go#L3002-L3008